components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/resources/repository/conf/identity/identity.xml

```diff
@@ -469,6 +469,7 @@
         <SAMLDefaultAssertionEncryptionAlgorithmURI>http://www.w3.org/2001/04/xmlenc#aes256-cbc</SAMLDefaultAssertionEncryptionAlgorithmURI>
         <SAMLDefaultKeyEncryptionAlgorithmURI>http://www.w3.org/2001/04/xmlenc#rsa-oaep-mgf1p</SAMLDefaultKeyEncryptionAlgorithmURI>
         <SLOHostNameVerificationEnabled>true</SLOHostNameVerificationEnabled>
+        <SAMLAuthnRequestsSignedEnabled>false</SAMLAuthnRequestsSignedEnabled>
```

Comment:
Make the config as "SAML2AuthnRequestsSigningEnabled" similar to https://github.com/wso2-support/carbon-identity-framework/pull/933/files#diff-58e3089af8f5a12e8b92c629111ca555R490